build.gradle

```diff
@@ -0,0 +1,24 @@
+buildscript {
+    repositories {
+        gradlePluginPortal()
+    }
+
+    dependencies {
+        classpath 'com.palantir.gradle.externalpublish:gradle-external-publish-plugin:1.2.1'
+        classpath 'com.palantir.gradle.gitversion:gradle-git-version:0.12.3'
+    }
+}
+
+apply plugin: 'com.palantir.external-publish-custom'
+apply plugin: 'com.palantir.git-version'
+
+group 'com.palantir.launching'
+
+version ((System.env.CIRCLE_TAG ?: gitVersion()).replaceAll('^v', ''))
+
+externalPublishing {
+    publication('dist') {
+        artifact file('out/dist/go-java-launcher/${project.version}/bin/go-java-launcher-${project.version}.tgz')
+        artifact file('out/dist/go-init/${project.version}/bin/go-init-${project.version}.tgz')
```

Comment:
Hmm, to publish two things I think we're going to need two publications, with different names

Reply:
```
# CIRCLE_TAG=1.12.0-10-g0ac11f6 ./gradlew publishToMavenLocal
> Task :publishDistPublicationToMavenLocal FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':publishDistPublicationToMavenLocal'.
> Failed to publish publication 'dist' to repository 'mavenLocal'
   > Invalid publication 'dist': multiple artifacts with the identical extension and classifier ('tgz.asc', 'null').
```